lib/node_modules/@stdlib/array/fixed-endian-factory/test/test.some.js

```diff
@@ -0,0 +1,189 @@
+	t.end();
+});
+
+tape( 'the method throws an error if invoked with a `this` context which is not a typed array instance', function test( t ) {
+	var values;
+	var ctor;
+	var arr;
+	var i;
+
+	ctor = factory( 'float64' );
+	arr = new ctor( 'little-endian', [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+
+	values = [
+		'5',
+		5,
+		NaN,
+		true,
+		false,
+		null,
+		void 0,
+		{},
+		[],
+		function noop() {}
+	];
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), TypeError, 'throws an error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			return arr.some.call( value, predicate );
+		};
+	}
+	function predicate( v ) {
+		return v < 0;
+	}
+});
+
+tape( 'the method throws an error if provided a first argument which is not a function', function test( t ) {
+	var values;
+	var ctor;
+	var arr;
+	var i;
+
+	ctor = factory( 'float64' );
+	arr = new ctor( 'little-endian', [ 1.0, 2.0, 3.0, 4.0 ] );
+
+	values = [
+		'5',
+		3.14,
+		NaN,
+		true,
+		false,
+		null,
+		void 0,
+		{},
+		[]
+	];
+	for ( i = 0; i < values.length; i++ ) {
+		t.throws( badValue( values[i] ), TypeError, 'throws an error when provided '+values[i] );
+	}
+	t.end();
+
+	function badValue( value ) {
+		return function badValue() {
+			return arr.some( value );
+		};
+	}
+});
+
+tape( 'the method returns `false` if operating on an empty array', function test( t ) {
+	var bool;
+	var ctor;
+	var arr;
+
+	ctor = factory( 'float64' );
+	arr = new ctor( 'little-endian' );
+	bool = arr.some( predicate );
+
+	t.strictEqual( bool, false, 'returns expected value' );
+	t.end();
+
+	function predicate() {
+		t.fail( 'should not be invoked' );
+	}
+});
+
+tape( 'the method returns `true` if any element pass a test', function test( t ) {
+	var bool;
+	var ctor;
+	var arr;
+
+	ctor = factory( 'float64' );
+	arr = new ctor( 'little-endian', [ -1.0, -2.0, 3.0, -4.0, 5.0 ] );
+	bool = arr.some( predicate );
+
+	t.strictEqual( bool, true, 'returns expected value' );
+	t.end();
+
+	function predicate( v ) {
+		return v > 0;
+	}
+});
+
+tape( 'the method returns `false` if all elements fail a test', function test( t ) {
+	var bool;
+	var ctor;
+	var arr;
+
+	ctor = factory( 'float64' );
+	arr = new ctor( 'little-endian', [ -1.0, -2.0, -3.0, -4.0, -5.0 ] );
+	bool = arr.some( predicate );
+
+	t.strictEqual( bool, false, 'returns expected value' );
+	t.end();
+
+	function predicate( v ) {
+		return v > 0;
+	}
+});
+
+tape( 'the method supports providing an execution context', function test( t ) {
+	var bool;
+	var ctor;
+	var arr;
+	var ctx;
+
+	ctor = factory( 'float64' );
+	arr = new ctor( 'little-endian', [ 1.0, 2.0, 3.0, 4.0, 5.0 ] );
+	ctx = {
+		'threshold': 3
+	};
+	bool = arr.some( predicate, ctx );
+
+	t.strictEqual( bool, true, 'returns expected value' );
+
+	t.end();
+
+	function predicate( v ) {
+		return v > this.threshold; // eslint-disable-line no-invalid-this
+	}
+});
```

Comment:
Similar to `array/bool` (see https://github.com/stdlib-js/stdlib/blob/28db4f2869749f6a6d6f7ae6476bd27f65052e96/lib/node_modules/%40stdlib/array/bool/test/test.some.js#L175), would you mind adding a test demonstrating that the predicate function is no longer called upon finding an element which passes a test?